RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+ComplexTypes.swift

```diff
@@ -359,6 +359,58 @@ extension OnLedgerEntitiesClient {
 		unstakeData: [NonFungibleGlobalId: UnstakeData],
 		tokens: [OnLedgerEntity.NonFungibleToken]
 	) throws -> KnownResourceBalance {
+		let stakeClaimTokens = try stakeClaimTokens(
```

Comment:
Is single stakeClaim used anywhere after your change?

Reply:
hmm, I think there we should also have it split. Strange to see it separated in TX Review, but aggregated in History. But let's confirm with Matt.